restless/tnd.py

```diff
@@ -128,6 +128,9 @@ def request_method(self):
     def request_body(self):
         return self.request.body 
 
+    def request_parameters(self):
+        return self.request.query_arguments
```

Comment:
This implementation is incomplete, [HTTPServerRequest.query_arguments](http://www.tornadoweb.org/en/stable/httputil.html#tornado.httputil.HTTPServerRequest.query_arguments) returns a mapping that points to `bytes`, not `str`. I'm not sure how to cleanly fix this, we could maybe use [RequestHandler.decode_argument](http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.decode_argument), but this would cause us to automatically parse the arguments before somebody actually needs them. Another option is to add a helper class around [RequestHandler.get_query_argument](http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.get_query_argument) with `__getitem__`